notebook/templates/edit.html

```diff
@@ -95,5 +109,12 @@
 
     {{super()}}
 
+<script>
+$(document).ready(function(){
+    $("#toggle_header").click(function(){
```

Comment:
You can put this code in MenuBar.bind_events in notebook/static/edit/js/menubar.js.

To see updates to the js while you are working, it's a good idea to run:

    npm run build:watch

in the notebook repo. That way, your changes will get recompiled when you make them.